syz-cluster/workflow/boot-step/main.go

```diff
@@ -76,11 +76,17 @@ func main() {
 // the test must fail 3 times in a row.
 const retryCount = 3
 
+// The base config may have more VMs, but we don't need that many.
+const vmCount = 3
```

Comment:
not important: silent parameters change may be confusing. Do we warn about this rewrite somehow?